spec/seal_spec.rb

```diff
@@ -10,18 +10,23 @@
       slack_channel: "#lions",
       quotes: ["go lions!"],
       quotes_days: ["Monday", "tuesday", "Wednesday"],
+      morning_seal_quotes: true,
```

Comment:
I'd fix this up to the first commit too - every commit should have a passing test suite, as a general rule. So I'm guessing commit 1 and 2 have a broken test suite until this point.